mender-convert-package

```diff
@@ -350,35 +351,31 @@ else
 fi
 mender_features="${bootloader_feature} mender-convert"
 
-cat <<- EOF > deploy/${image_name}.cfg
-MENDER_BOOT_PART="${boot_part_device}"
-MENDER_ROOTFS_PART_A="${root_part_a_device}"
-MENDER_ROOTFS_PART_B="${root_part_b_device}"
-MENDER_DATA_PART="${data_part_device}"
-MENDER_BOOT_PART_MOUNT_LOCATION="${boot_part_mountpoint}"
-MENDER_BOOT_PART_SIZE_MB="$(disk_sectors_to_mb ${boot_part_sectors})"
-MENDER_DATA_PART_SIZE_MB="${MENDER_DATA_PART_SIZE_MB}"
-MENDER_DEVICE_TYPE="${device_type}"
-MENDER_PARTITIONING_OVERHEAD_KB="$(( (${overhead_sectors} * 512) / 1024 ))"
-MENDER_PARTITION_ALIGNMENT="${MENDER_PARTITION_ALIGNMENT}"
-MENDER_STORAGE_TOTAL_SIZE_MB="${MENDER_STORAGE_TOTAL_SIZE_MB}"
-MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET="12582912"
-MENDER_ARTIFACT_NAME="${artifact_name}"
-MENDER_FEATURES="${mender_features}"
-DEPLOY_DIR_IMAGE="${PWD}/deploy"
-MENDER_MACHINE="${device_type}"
-EOF
-
+testscfg_add "MENDER_BOOT_PART" "${boot_part_device}"
+testscfg_add "MENDER_ROOTFS_PART_A" "${root_part_a_device}"
+testscfg_add "MENDER_ROOTFS_PART_B" "${root_part_b_device}"
+testscfg_add "MENDER_DATA_PART" "${data_part_device}"
+testscfg_add "MENDER_BOOT_PART_MOUNT_LOCATION" "${boot_part_mountpoint}"
+testscfg_add "MENDER_BOOT_PART_SIZE_MB" "$(disk_sectors_to_mb ${boot_part_sectors})"
+testscfg_add "MENDER_DATA_PART_SIZE_MB" "${MENDER_DATA_PART_SIZE_MB}"
+testscfg_add "MENDER_DEVICE_TYPE" "${device_type}"
+testscfg_add "MENDER_PARTITIONING_OVERHEAD_KB" "$(( (${overhead_sectors} * 512) / 1024 ))"
+testscfg_add "MENDER_PARTITION_ALIGNMENT" "${MENDER_PARTITION_ALIGNMENT}"
+testscfg_add "MENDER_STORAGE_TOTAL_SIZE_MB" "${MENDER_STORAGE_TOTAL_SIZE_MB}"
+testscfg_add "MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET" "12582912"
+testscfg_add "MENDER_ARTIFACT_NAME" "${artifact_name}"
+testscfg_add "MENDER_FEATURES" "${mender_features}"
+testscfg_add "DEPLOY_DIR_IMAGE" "${PWD}/deploy"
+testscfg_add "MENDER_MACHINE" "${device_type}"
 
 # Outputting device base only relevant for some configurations
```

Comment:
Q:
Still only for some configurations?

Reply:
A:
Only `if [ "${MENDER_ENABLE_PARTUUID}" != "y" ]`, right? I dunno, it was there and I haven't changed the if condition neither...